startup/buildInfo.js

```diff
@@ -159,6 +159,28 @@ function dockerfileBuilder(appConfig, tags) {
     };
 }
 
+// This is the same command as nixpacks generates except it uses
+// php-fpm -D instead of backgrounding via shell ("&"). This allows
+// php-fpm to start and background itself so that it is ready to
+// accept connections when nginx is started.
+const startPhpScript = `#!/usr/bin/env bash
+node /assets/scripts/prestart.mjs /assets/nginx.template.conf /nginx.conf && (php-fpm -D -y /assets/php-fpm.conf && nginx -c /nginx.conf)
```

Comment:
So how would a user use this to do a DB migration?

Reply:
You'll see that in the docs updates. But they'd set Start Command to:

```
php artisan migrate --force && start-php
```

Reply:
OK, that makes sense